react/features/base/tracks/middleware.js

```diff
@@ -159,18 +159,19 @@ function _getLocalTrack(store, mediaType: MEDIA_TYPE) {
  */
 function _setMuted(store, action, mediaType: MEDIA_TYPE) {
     const localTrack = _getLocalTrack(store, mediaType);
+    const muted = Boolean(action.muted);
 
     if (localTrack) {
-        setTrackMuted(localTrack.jitsiTrack, action.muted)
+        setTrackMuted(localTrack.jitsiTrack, muted)
             .catch(error => {
-                console.error(`setTrackMuted(${action.muted}) failed`, error);
-
-                const setMuted
-                    = mediaType === MEDIA_TYPE.AUDIO
-                        ? setAudioMuted : setVideoMuted;
+                console.error(`setTrackMuted(${String(muted)}) failed`, error);
 
                 // Failed to modify muted state - dispatch rollback action
-                store.dispatch(setMuted(!action.muted));
+                if (mediaType === MEDIA_TYPE.AUDIO) {
+                    store.dispatch(setAudioMuted(!muted));
+                } else {
+                    store.dispatch(setVideoMuted(!muted, action.muteType));
```

Comment:
1. There's no action with a property named `muteType`. It's not because I renamed `VIDEO_MUTE_TYPE` and related variables.

2. This PR modifies a carbon copy of a piece of source code. Copy-pasting is a violation of our written coding style.

3. Now the two copies are diverging which is another problem.